py/dml/codegen.py

```diff
@@ -2087,19 +2110,24 @@ def stmt_switch(stmt, location, scope):
 
 @statement_dispatcher
 def stmt_continue(stmt, location, scope):
-    if not loop_stack or loop_stack[-1] == 'method':
+    if not loop_stack or isinstance(loop_stack[-1], NoLoopContext):
         raise ECONT(stmt.site)
-    elif loop_stack[-1] == 'c':
+    elif isinstance(loop_stack[-1], CLoopContext):
         return [mkContinue(stmt.site)]
     else:
         raise ECONTU(stmt.site)
```

Comment:
It would make sense to permit `continue` as well, for symmetry. Not strictly needed to be part of this patch though.

Reply:
`continue` would clutter up generated code with a lot of `goto`s and labels; and besides, unlike `break`, there are easy usages of `if`-`else` that can be used instead of `continue`. For those reasons, I'm not as keen on supporting `continue`.

Reply:
Fair enough. I don't have a problem with procrastinating `continue` until someone asks for it.